.changeset/small-cameras-tickle.md

```diff
@@ -0,0 +1,7 @@
+---
+"@equinor/mad-components": patch
+---
+
+Updated search. Search bar can now clear text. Added optional cancel button. Fixed flicker bug on
```

Comment:
Did the old component behave like this (flickering)? 
If you are reffering to Notifications, the search bar they use there is not mad components, so in that case, this last sentence can be removed